WireUI/Sources/WireAccountImageUI/AccountImageView/AccountImageView.swift

```diff
@@ -232,9 +236,25 @@ public final class AccountImageView: UIView {
             accountImageViewWrapper.bottomAnchor.constraint(equalTo: availabilityIndicatorView.bottomAnchor)
         ])
 
+        // view which renders the notification badge
+        notificationBadgeView.translatesAutoresizingMaskIntoConstraints = false
+        addSubview(notificationBadgeView)
+        NSLayoutConstraint.activate([
+            notificationBadgeView.widthAnchor.constraint(
+                equalToConstant: 14
+            ),
+            notificationBadgeView.heightAnchor.constraint(equalToConstant: 14),
+            notificationBadgeView.centerYAnchor.constraint(equalTo: accountImageViewWrapper.topAnchor, constant: 3.5),
+            notificationBadgeView.centerXAnchor.constraint(
+                equalTo: accountImageViewWrapper.trailingAnchor,
+                constant: -3.5
+            )
```

Comment:
Could you try to make the size of the dot relative, please?
I think the remaining code somehow scales properly. (The view is not always displayed in the same size, like between iPhone and iPad [sidebar])
![ezgif-4-a979cb42f5](https://github.com/user-attachments/assets/401eb97d-d334-477b-bd31-f41585ffb321)

Reply:
Actually the border around the image doesn't seem to scale, but still, it would be nice.